source/YugiAcademy/YuGiOh/booster_packs/tests/test_booster_pack_card.py

```diff
@@ -67,13 +75,21 @@ def test_booster_pack_card_identifier_must_be_related_to_booster_pack_code():
 @pytest.mark.django_db
 def test_booster_pack_card_instance_creation_and_accessing():
     for card in [pot_of_greed(), jar_of_greed(), dark_magician()]:
-        pack = booster_pack()
+        booster_pack = legend_of_blue_eyes_white_dragon()
+        identifier = 'LOB-EN119'
+        rarity = 'Rare'
         booster_pack_card = BoosterPackCard.referring_to(card=card,
-                                                         booster_pack=pack,
-                                                         identifier='LOB-EN119',
-                                                         rarity='Rare')
-        assert booster_pack_card.card == card
-        assert booster_pack_card.card_name() == card.name
-        assert booster_pack_card.booster_pack == pack
-        assert booster_pack_card.identifier == 'LOB-EN119'
-        assert booster_pack_card.rarity == 'Rare'
+                                                         booster_pack=booster_pack,
+                                                         identifier=identifier,
+                                                         rarity=rarity)
+        assert_is_expected(booster_pack_card, card, booster_pack, identifier, rarity)
+
+
+@pytest.mark.django_db
+def test_booster_pack_card_instance_creation_from_form():
+    for card in [pot_of_greed(), jar_of_greed(), dark_magician()]:
+        booster_pack = legend_of_blue_eyes_white_dragon()
+        identifier = 'LOB-EN119'
+        rarity = 'Rare'
+        booster_pack_card = BoosterPackCard.from_form(locals())
```

Comment:
It's not descriptive what locals() is... I don't understand what it is

Reply:
locals() is a python function that makes a dictionary with the local variables as keys and their values as... yeah, values! It's useful in this test since I don't want to make the dictionary by myself.